File: Data Structures & Algorithms/single-row-keyboard/submission-0.py
class Solution:
    def calculateTime(self, keyboard: str, word: str) -> int:
        curr_idx = 0
        res = 0
        for c in word:
            for k in range(len(keyboard)):
                if c == keyboard[k]:
                    res += abs(curr_idx-k)
                    curr_idx = k
        

        return res

File: Data Structures & Algorithms/valid-word-square/submission-0.cpp
class Solution {
public:
    bool validWordSquare(vector<string>& words) {
        n = words.length();
        int row_length = 0;
        for(int i=0;i<n;i++){
            row_length = word[i].length();
            if(i != 0 && row_length != word[i].length()){
                return false;
            }
        }

        return true;
    }
};
